README.md

```diff
@@ -65,27 +87,92 @@ The Dataset can be implemented with various librairies, following the implicit c
 * ``data/raw`` stores the raw data files;
 * ``data/processed`` stores the data after preprocessing, used for several experiments.
 
-## Quick Start
+## Experiment a model project
 
-Each model can be experimented in an easy and quick way using predifined command exposed through the Nox tool.
+### Setting-up the environment
 
-### Docker
+Each model can be experimented using a python environment dedicated to the project, either in a docker container or on bare metal using a virtual environment.
 
-A docker file is provided to get started. To build the image, run the following command:
-::
-    docker build -t ai4sim .
+#### Docker container
 
-To install the corresponding python requirements for a use-case an entrypoint is implemented. It can be selected by one of the folling [combustion_gnns, combustion_unets, wf_gwd]:
-::
-    docker run -it --rm ai4sim ./script.sh combustion_gnns
+The AI4SIM github CI/CD publishes in the [github registry](https://github.com/AI4SIM/model-collection/pkgs/container/model-collection) a docker image dedicated to each model project proposed in the **Model collection** repository. Each image is built on an public **Ubuntu** base image (e.g. *nvidia/cuda:11.7.1-cudnn8-runtime-ubuntu20.04*). All the model project's requirements have been installed and the model project code has been added in ``/home/ai4sim/<domain>/<use-case>/<NN architecture>``.
```

Comment:
The model project code has been added in ``/home/ai4sim/<domain>/<use-case>/<NN architecture>`` and its requirements have been installed in the container